flashlight/lib/CMakeLists.txt

```diff
@@ -2,23 +2,9 @@ cmake_minimum_required(VERSION 3.10)
 
 # ------------------------- Library -------------------------
 
-add_library(
-  fl-libraries
-  ""
-  )
-set(INSTALLABLE_TARGETS ${INSTALLABLE_TARGETS} fl-libraries)
-
-target_include_directories(
-  fl-libraries
-  PRIVATE
-  ${PROJECT_SOURCE_DIR}
-  )
-
-target_compile_definitions(
-  fl-libraries
-  PUBLIC
-  ${FL_COMPILE_DEFINITIONS}
-  )
+set(INSTALLABLE_TARGETS ${INSTALLABLE_TARGETS} fl_lib_audio)
```

Comment:
optional nit: can we move these closer to the actual `add_library(fl_lib_**** ...` in each lib component's respective directory? feel free to ignore

Reply:
Sure thing.